bin/ci/az-publish.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# $1 => SOURCE PATH
+# $2 => TARGET NAME
+function azupload {
+  az storage blob upload \
+    --container-name test-apps \
+    --file "${1}" \
+    --name "${2}"
+  echo "${1} artifact uploaded with name ${2}"
+}
+
+function info {
+  if [ "${TERM}" = "dumb" ]; then
+    echo "INFO: $1"
+  else
+    echo "$(tput setaf 2)INFO: $1$(tput sgr0)"
+  fi
+}
+
+function zip_with_ditto {
+  xcrun ditto \
+  -ck --rsrc --sequesterRsrc --keepParent \
+  "${1}" \
+  "${2}"
+  info "Installed ${2}"
+}
+
+# Pipeline Variables are set through the AzDevOps UI
+# See also the ./azdevops-pipeline.yml
+if [ -e ./.azure-credentials ]; then
+  source ./.azure-credentials
+fi
+
+if [[ -z "${AZURE_STORAGE_ACCOUNT}" ]]; then
+  echo "AZURE_STORAGE_ACCOUNT is required"
+  exit 1
+fi
+
+if [[ -z "${AZURE_STORAGE_KEY}" ]]; then
+  echo "AZURE_STORAGE_KEY is required"
+  exit 1
+fi
+
+if [[ -z "${AZURE_STORAGE_CONNECTION_STRING}" ]]; then
+  echo "AZURE_STORAGE_CONNECTION_STRING is required"
+  exit 1
+fi
+
+WORKING_DIR="${BUILD_SOURCESDIRECTORY}"
+PRODUCT_DIR="${WORKING_DIR}/CalSmokeApp/Products/app/CalSmoke-cal"
```

Comment:
Since we use both `app` and `ipa` folder, maybe it makes sense to define and use these kind of variables: `IPA_PRODUCTS_DIR`, `APP_PRODUCTS_DIR`?